addons/web/static/src/js/widgets/auto_complete.js

```diff
@@ -51,6 +52,11 @@ return Widget.extend({
                 }
                 return;
             }
+        });
+        this.$input.on('input', function (ev) {
+            if (self._isInputComposing && !config.device.isMobile) {
```

Comment:
no autocomplete on desktop anymore?

Reply:
The composing stuff has 2 meanings:

1. On desktops, it means you're using combination keys. E.g. hit <kbd>¨</kbd> <kbd>u</kbd> to get ü.
2. On mobiles, it means you're using AI completion in your on-screen keyboard, or you're composing a word that might get corrected by the spell checker.

If you're composing on a computer, you don't want autocompletion enabled usually until you finish composing the desired character.